pkg/manifest/yaml/resources.go

```diff
@@ -0,0 +1,59 @@
+package yaml
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+// WriteResources takes a prefix path, and a map of paths to values, and will
+// marshal the values to the filenames as YAML resources, joining the prefix to
+// the filenames before writing.
+//
+// It returns the list of filenames written out.
+func WriteResources(path string, files map[string]interface{}) ([]string, error) {
+	filenames := make([]string, 0)
+	for filename, item := range files {
+		err := marshalItemsToFile(filepath.Join(path, filename), list(item))
```

Comment:
We can write/marshal a single item (and get rid of list()).  (An item can be a single structure, slice, or map, etc).
`err := marshalItemToFile(filepath.Join(path, filename), item)`